frontend/src/features/properties/map/MapRouter.tsx

```diff
@@ -91,9 +91,14 @@ export const MapRouter: React.FunctionComponent<IMapRouterProps> = React.memo(pr
         title={'Property Information'}
       />
       <AppRoute
-        path={`/mapview/sidebar/non-inventory-property/:pid`}
+        path={`/mapview/sidebar/property/:id/non-inventory-property/:pid`}
```

Comment:
This doesn't make sense unfortunately.

non-inventory-property means that the property is not in our database. As a result, the property:id must always be null. If we are being navigated to this url when a property is in our inventory something is wrong. This route should only be used when we are viewing a property that is not in our inventory on the map.

Reply:
What should be happening here is the frontend should be smart enough to either route to the regular property endpoint: (ie. mapview/property/24324) when the property is in our inventory, and to /mapview/sidebar/non-inventory-property/:pid when the property is not in our inventory.

If it is in our inventory we should have an id and a marker on the map.